src/tile-board/Tile.jsx

```diff
@@ -1,11 +1,26 @@
 import React from "react";
+import propTypes from "prop-types";
+import "./Tile.css";
 
-function Tile() {
+function Tile({ isOn, id, handleToggle }) {
   return (
-    <button type="button" className="btn btn-light btn-outline-dark">
-      Tile
-    </button>
+    <button
+      type="button"
+      className={
+        isOn
+          ? "btn btn-light btn-outline-dark tile tile-on"
+          : "btn btn-dark btn-outline-light tile tile-off"
+      }
+      id={id}
+      onClick={handleToggle}
+    ></button>
```

Comment:
[Here](https://www.w3.org/WAI/standards-guidelines/act/rules/97a4e1/)'s a bit of a technical reference to the a11y rule around accessible buttons. Though it's hard to imagine this game being totally keyboard accessible, it's worth adding some screen-reader only text so that these buttons aren't "nameless".